platforms/gcp-dataproc-serverless-airgapped/spark_healthcare_nlp_serverless.py

```
@@ -0,0 +1,77 @@
+import json, os
+
+import sparknlp
```

Ruff (0.11.9)

`sparknlp` imported but unused

Remove unused import: `sparknlp`

(F401)